ccmlib/cmds/node_cmds.py

```diff
@@ -459,6 +460,40 @@ def run(self):
                                    column_families=self.column_families, size=self.size,
                                    no_snapshot=self.no_snapshot)
 
+class NodeGetsstablesCmd(Cmd):
+    def description(self):
+        return "Run getsstables to get sstables of this node"
+
+    def get_parser(self):
+        usage = "usage: ccm node_name getsstables [options] [file]"
+        parser = self._get_default_parser(usage, self.description())
+        parser.add_option('-k', '--keyspace', type="string", dest="keyspace", default=None,
+                          help="The keyspace to use [use all keyspaces by default]")
+        parser.add_option('-c', '--column-families', type="string", dest='cfs', default=None,
+                          help="Comma seperated list of column families ut use (requires -k to be set)")
```

Comment:
This one I just picked from NodeSstablesplitCmd(Cmd) so didn't look much at the grammar. Have corrected that too now.